MAPL_Base/MAPL_EtaHybridVerticalCoordinate.F90

```diff
@@ -0,0 +1,282 @@
+#include "MAPL_Generic.h"
+#include "MAPL_ErrLog.h"
+#include "unused_dummy.H"

```

Comment:
I think only the middle #include is needed here.

Reply:
we only need the first one